Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.10.4"
 [deps]
 Ipopt = "b6b21f68-93f8-5de0-b562-5493be1d77c9"
 NLPModels = "a4795742-8479-5a88-8948-cc11e1c8c1a6"
+NLPModelsModifiers = "e01155f1-5c6f-4375-a9d8-616dd036575f"
```

Comment:
You need to add the version number in the compat section when adding a new dependency